src/Pos/Explorer/DB.hs

```diff
@@ -28,6 +30,17 @@ import           Pos.Txp.Toil          (Utxo)
 import           Pos.Util.Chrono       (NewestFirst (..))
 
 ----------------------------------------------------------------------------
+-- Types
+----------------------------------------------------------------------------
+
+type Page = Integer
```

Comment:
Ok, I don't mind having this alias, but probably it can be bounded (e. g. `Int64`)? I don't think it will ever be greater than 10¹⁸.